### ISSUE TITLE
Feature request: Make @+ mapping configurable

### ISSUE THREAD

**Issue by zell-mbc** [rkokkelk/todo.txt-ext#57]
After loking at https://github.com/todotxt/todo.txt I think the mapping looks like below:
todotxt   "+" = project tag    "@" = Context tag
AddOn   "+" = category       "@" = Location

Using one of their examples, "(B) Schedule Goodwill pickup +GarageSale @phone" I would say the mapping should be the other way round?
I use Simpletask on my Android phone to get to the todo.txt which seems to handle things the opposite way as well.
However, you may have had good reasons to do it the way you did, so how about making the mapping configurable?

**Comment by Bellamonte** [comment]
I guess, as Thunderbird/Lightning doesn't have a field for neither Project nor Context. That's why the extension by default uses fields Category and Location, respectively.
If you choose to not use Project tag as Category and Context tag as Location, there's no other way as to show both tags inside description of the Task in Thunderbird/Lightning.
There's even an option for not having above stated default behaviour in the extension's settings, so the tags will be included in the description.

I would say, this is not a bug then...

**Comment by zell-mbc** [comment]
I agree, this is not a bug. More like a feature request. All I was asking if we configure what goes where to allow this add-on to work with other tools (in my case Simpletask on Android) which store this information the other way round.

**Comment by JoergPalmer** [comment]
I tend to agree with @zell-mbc ... it is confusing. Funny enough, if you dare to edit a task in Thunderbird, it clearly labels what is called a "project" (in this extensions' context) as "Category". To me using that for "context" (in the GTD sense) sounds like a better choice. Why is that? You can flag categories (in TB sense) with color tags. This is very helpful for organizing the calendar and task views.
